go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/fortnoxab/bitbucket-slack-bot
 
-go 1.17
+go 1.18
```

Comment:
Suggested update in order to use the `any` type in the test helper `expect` function.